content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-oss.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Monitor InfluxDB Enterprise using OSS 
```

Comment:
This doc needs further work to change Cloud-related steps to OSS 2.0.

Reply:
Finished the changes!